### DISCUSSION TITLE
Does Lava conform to UWP windows policy ?

### DISCUSSION THREAD

**Discussion by poupa29** [lava-nc/lava#281] [Q&A] [answered]
Hi,
To be used on Windows mobile devices, Lava should be deployed as an UWP-conform app (Universal Windows Platform). Is that already the case, or is there a plan to?

Regards, Axel

**Comment by awintel** [comment, marked as answer]
At Intel, we have not looked into this and have no plan doing so yet. 
Could you elaborate on use cases, what it would take and whether you are interested in contributing to such a possible future effort?

**Reply by poupa29** [reply]
In the frame of a project on a windows edge computer, either we will implement it or we will use a proxy rasperry pi with Lava installed. Thanks for the information.
